test/helpers/policygen/actions.go

```diff
@@ -0,0 +1,69 @@
+package policygen
+
+import (
+	"fmt"
+
+	"github.com/cilium/cilium/test/helpers"
+	log "github.com/sirupsen/logrus"
+)
+
+// HTTPAction runs a helpers.CurlFail from specified pod to a specified target.
+// It needs a `helpers.Kubectl` instance to run the comamnd in the pod. It
+// returns a ResultType struct.
+func HTTPAction(srcPod string, target string, kub *helpers.Kubectl) ResultType {
+	command := fmt.Sprintf("%s exec -n %s %s -- %s",
+		"kubectl", helpers.DefaultNamespace,
```

Comment:
Make the `kubectl` in the `test/helpers` package public and use it here instead of hardcoding this string.